src/process.h

```diff
@@ -167,8 +167,13 @@ class Process : public ProcessListFromProcessGroup::Element,
   void set_target_priority(uint8 value) { target_priority_ = value; }
   uint8 update_priority();
 
+#if defined(TOIT_WINDOWS)
+  const char* current_directory();
```

Comment:
Maybe it would be cleaner to just have separate names for this field? It is handled separately everywhere.

Maybe cwd_path_ and cwd_fd_?

Reply:
Yes.
On the other hand, it would add an extra unused word per process on embedded systems.

Reply:
I'm not suggesting you have both fields on all platforms!

Reply:
So, your suggestion is to keep the #def's as is but have different field names? I think the CWD could be relevant on FreeRTOS too when the flash PR is merged.